package.json

```diff
@@ -30,30 +30,29 @@
   "homepage": "https://wwebjs.dev/",
   "dependencies": {
     "@pedroslopez/moduleraid": "^5.0.2",
-    "fluent-ffmpeg": "2.1.3",
-    "mime": "^3.0.0",
-    "node-fetch": "^2.6.9",
-    "node-webpmux": "3.1.7",
-    "puppeteer": "^18.2.1"
+    "fluent-ffmpeg": "^2.1.3",
+    "mime": "^4.0.7",
+    "node-fetch": "^3.3.2",
```

Comment:
Here as in node-fetch > v3.x.x and mime > v4.x.x is no longer supports requiring modules in commonjs, such as using require(), you should use the import() instead.
More information about node-fetch: [read here](https://github.com/node-fetch/node-fetch#commonjs)
Also about mime: [read here](https://github.com/broofa/mime#mime)

Because whatsapp-web.js still uses [commonjs](https://nodejs.org/api/modules.html#modules-commonjs-modules) modularity, there will be a problem of differences in version support.